examples/js/web-elements-app/src/app/app.element.ts

```diff
@@ -24,6 +31,8 @@ export class AppElement extends HTMLElement {
           <h1>
             <span> Hello there, </span>
             ${titleVariable.value}
+            <br><br>
+            <span> EnableEdgeDB status: ${enableStatus} </span>
```

Comment:
is this meant to indicate that the 'cloud-data-demo' flag is enabled due to some data we got from the edgedb? Or is it saying that the edgedb itself is enabled?

If it's the former, you may want to change the wording here to say something like "Demo Flag Status"